tools/src/h5repack/h5repack_opttable.c

```diff
@@ -44,47 +44,47 @@ init_packobject(pack_info_t *obj)
 /*-------------------------------------------------------------------------
  * Function: aux_tblinsert_filter
  *
- * Purpose: auxiliary function, inserts the filter in object OBJS[ I ]
+ * Purpose: auxiliary function, inserts the filter in object OBJS[ idx ]
  *
  * Return: void
  *-------------------------------------------------------------------------
  */
 
 static void
-aux_tblinsert_filter(pack_opttbl_t *table, unsigned int I, filter_info_t filt)
```

Comment:
`I` had to be changed since it is a keyword from the complex.h header and conflicted.